rclcpp/test/rclcpp/test_subscription.cpp

```diff
@@ -461,6 +461,33 @@ TEST_P(TestSubscriptionInvalidIntraprocessQos, test_subscription_throws) {
   }
 }
 
+/*
+   Testing subscription with invalid use_intra_process_comm
+ */
+TEST_P(TestSubscriptionInvalidIntraprocessQos, test_subscription_throws_intraprocess) {
+  // Adds more coverage
```

Comment:
I don't think we need this comment; the one above the test seems to spell out what it does.